docs/self-hosted-appcircle/install-server/helm-chart/installation/docker-k8s-migration.md

```diff
@@ -0,0 +1,1343 @@
+
+- Creating a VM snapshot of the Docker host.
+- Using a backup tool specific to your infrastructure.
+- Ensuring configuration files and database dumps are included in your backups.  
+  :::
+
+#### 1. Create a Migration Directory
+
+On your **bastion host**, create a directory to store all migration files:
+
+```bash
+mkdir appcircle-k8s-migration && cd appcircle-k8s-migration
+```
+
+#### 2. Backup Appcircle Configuration Data
+
+On the **standalone Appcircle server**, execute the following commands to back up the necessary files. Transfer the outputs to the migration directory on the bastion host.
+
+- **Appcircle Server Directory:**
+  Change directory to the `appcircle-server`.
+  ```bash
+  cd appcircle-server
+  ```
+  <SpacetechExampleInfo/>
+- **`global.yaml` Configuration:**
+  Print the `global.yaml` file and save it on the bastion host:
+
+  ```bash
+  cat projects/spacetech/global.yaml
+  ```
+
+- **User Secrets:**
+  Decode and print the `user-secret` file and save it on the bastion host:
+
+  ```bash
+  cat projects/spacetech/user-secret | base64 -d
+  ```
+
+- **Generated Secrets:**
+  Print the `generated-secret.yaml` file and save it on the bastion host:
+
+  ```bash
+  cat projects/spacetech/generated-secret.yaml
+  ```
+
+- **`cred.json` file:**
+  Print the `cred.json` file and save it on the bastion host:
+  ```bash
+  cat cred.json
+  ```
+
+#### 3. Check the Data Size on the Standalone Appcircle Server
+
+- **Get the volume sizes:**
+
+  ```bash
+  export APPCIRCLE_DISK_USAGE=$(docker system df -v)
+  ```
+
+- **Check the MinIO data size:**
+
+  ```bash
+  echo "$APPCIRCLE_DISK_USAGE" | grep "spacetech_snsd_data"
+  ```
+
+- **Check the MongoDB data size:**
+
+  ```bash
+  echo "$APPCIRCLE_DISK_USAGE" | grep "spacetech_mongo_data"
+  ```
+
+- **Check the PostgreSQL data size:**
+
+  ```bash
+  echo "$APPCIRCLE_DISK_USAGE" | grep "spacetech_posgresqlData"
+  ```
+
+- **Check the Vault data size:**
+  ```bash
+  echo "$APPCIRCLE_DISK_USAGE" | grep "spacetech_vault_data"
+  ```
+
+You will use those values while configuring the `values.yaml` of the Appcircle server Helm chart.
+
+#### 4. Make Sure the Standalone Appcircle Server Is Running
+
+Before proceeding with the migration, verify that the standalone Appcircle server is operational and healthy. This ensures you can create accurate backups without issues.
+
+- **Log in to the standalone Appcircle server:**
+
+- **Change directory to the `appcircle-server`:**
+
+   ```bash
+   cd appcircle-server
+   ```
+
+- **Check the health status of the standalone Appcircle server:**
+   <SpacetechExampleInfo/>
+
+   ```bash
+   ./ac-self-hosted.sh -n spacetech check
+   ```
+
+#### 5. Update the Standalone Appcircle Server to the Latest Version
+
+Before migrating to Kubernetes Appcircle server, please [update the standalone Appcircle server](https://docs.appcircle.io/self-hosted-appcircle/install-server/linux-package/update) to the latest version.
```

Comment:
Migration is supported for 3.23.2 or later Appcircle servers.

If that's OK, then user should select a helm chart latest version that's compatible with his/her standalone server.

We should have link to chart/app version history.